src/app/user-utils/user-service.ts

```diff
@@ -15,10 +16,11 @@ export class UserService {
     }
 
     public getOwnProfile() {
-        const userResponse = this.http.get(API.generateRoute(this.endpoint), API.tokenHeaders());
-        userResponse.subscribe((res: UserModel) => {
-            UserStatusStorage.personalData = res;
-        });
-        return userResponse;
+        return this.http.get<UserModel>(API.generateRoute(this.endpoint), API.tokenHeaders())
+            .pipe(
+                map(res => {
+                    UserStatusStorage.personalData = res;
+                    return res;
+                }));
```

Comment:
Try using `tap` operator so you don't need to `return res` again. `tap` will return the same value as the given one in the input.

Reply:
Fixed!